src/commands/slash/suggest.ts

```diff
@@ -41,37 +42,24 @@ export const command = new SlashCommand(
 
 			const embed: APIEmbed = {
 				title: `New Suggestion! #${lastSuggestionId}`,
-				description: `${interaction.options.getString("suggestion", true)}`,
+				description: `${interaction.options.getString(
+					"suggestion",
+					true
+				)}\n\n_Suggested by ${interaction.user.tag}_`,
 				footer: {
-					text: `Suggested by ${interaction.user.tag}`,
+					text: `Upvotes : 0 | Downvotes : 0`,
 					icon_url: interaction.member.displayAvatarURL(),
 				},
 				color: config.commandsEmbedsColor,
 			};
 
-			const row = new MessageActionRow().addComponents(
-				{
-					customId: "suggestion-accept",
-					disabled: false,
-					emoji: "✅",
-					label: "Accept",
-					style: "SUCCESS",
-					type: "BUTTON",
-				},
-				{
-					customId: "suggestion-decline",
-					disabled: false,
-					emoji: "❌",
-					label: "Decline",
-					style: "DANGER",
-					type: "BUTTON",
-				}
-			);
-			await channel.send({
+			const suggestionMessage = await channel.send({
 				embeds: [embed],
-				components: [row],
 			});
 
+			await suggestionMessage.react("👍");
+			await suggestionMessage.react("👎");
```

Comment:
This is so inefficient while we can queue them at the same time...
```suggestion
			await Promise.all(suggestionMessage.react("👍"), suggestionMessage.react("👎"));
```